doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -169,6 +169,115 @@ Create OpenLayers application
 
 These tiles are being rendered by the OpenLayers client.
 
+Create OpenLayers application - WMS Vector Tiles
+------------------------------------------------
+
+.. note::   Vector tiles requested with WMS allows to retrieve non-cached vector tiles (server side), setting the ``tiled=false`` parameter on the ``getMap`` request. This setting could be particularly useful when serving fast changing source data that should constantly be kept up-to-date for display.
+
+#. Create a ``www/wms-vectortiles`` directory inside your GeoServer Data Directory.
+
+#. Download the `latest version of OpenLayers <http://openlayers.org/download/>`_.
+
+#. Extract the following files to from the downloaded archive to the directory created in step 1:
+
+   * ``ol.js``
+   * ``ol-debug.js``
+   * ``ol.css``
+
+#. In a text editor, create a new file with the following content:
+
+   .. code-block:: html
+      :emphasize-lines: 5
+      
+      <!doctype html>
+      <html>
+      <head>
+        <title>Vector tiles</title>
+        <script src="ol.js"></script>
+        <link rel="stylesheet" href="ol.css">
+        <style>
+          html, body {
+            font-family: sans-serif;
+            width: 100%;
+          }
+          .map {
+            height: 500px;
+            width: 100%;
+          }
+        </style>
+      </head>
+        <body>
```

Comment:
Looks like the formatting is off. Possibly reformat it before incorporating it?

Reply:
yes it is off